lnwallet/btcwallet/btcwallet.go

```diff
@@ -690,6 +700,19 @@ func (b *BtcWallet) ListAccounts(name string,
 	return res, nil
 }
 
+// newAccountNotFoundError returns an error indicating that the manager didn't
+// find the specific account. This error is used to be compatible with the old
+// 'LookupAccount' behaviour previously used.
+func newAccountNotFoundError(name string) error {
```

Comment:
I used this particular error to be compatible with the old error returned by LookupAccount in the same case.

If this compatibility is not important, I will create a generic error like
var errAccountNotFound